.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+<!--
+	Please populate this Pull Request template for **every** pull request
+	unless there is a really good reason against it. Your reviewer as well
+	as your future self will be thankful for the context given.
+
+	If you have specific code level changes to discuss, it's also helpful to
+	leave comments about your concerns at line level before requesting review.
+-->
+
+## Link to Phabricator
```

Comment:
I would personally favour keeping this in sync with gerrit etc. which suggests ending the message with a line such as:

`Bug: T12344`